### PR TITLE
Using HN's API and enabeling relative stats

### PR DESCRIPTION
**Pull request by devndive**
Fixing #12

### CODE REVIEW

File: src/api/fetchNewlyAddedHTMLs.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const fs = require('fs')
+const path = require('path')
+const axios = require('axios')
+
+const apiBase = 'https://hacker-news.firebaseio.com/v0/item'
+
+async function fetchNewlyAddedHTMLs () {
+  const baseFolder = path.join(__dirname, '../../assets/api')
```
Comment (timqian) [inline src/api/fetchNewlyAddedHTMLs.js]: seems path should be `../../assets/api/processsed`?

**Comment by timqian** [comment]
It seems the script did not work
```bash
node src/api/index.js react
```
screen shot 2019-02-26 at 12 21 41 pm

the reason seems be caused by the wrong path in `index.js`

Also the relative img seems wrong too.

screen shot 2019-02-26 at 12 36 27 pm

Also, I think using API is a better idea, so we don't need to keep the old contents in github & npm (user will need to download them if they want to use). So it will be great if these two things is done in this PR
- remove old implementation and assets
- update sample graphs in readme

Thank you very much~

**Comment by devndive** [comment]
Thanks for the review! I will remove the old implementation and assests and update the README.md.

**Comment by timqian** [comment]
Thank you very much @devndive This update is awsome!

**Pull request merged**